common/src/main/java/com/mx/path/core/common/connect/Request.java

```diff
@@ -285,10 +286,6 @@ public final void setHeaders(MultiValueMappable<String, String> singleValueMap)
     this.headers = new SingleValueMap<>(singleValueMap);
   }
 
-  public final void setTimeout(Duration timeout) {
```

Comment:
Isn't this a breaking change? You could leave it and just have it set this.requestTimeout. Then mark it as deprecated.

Reply:
This is technically a breaking change, but a newly added function so it is safe to assume nobody pulled this in to use in the last 10 minutes. I added this when I thought we were going to preserve usage of `timeOut`. See that I have deprecated `setTimeOut`

Reply:
Not a good precedent to set. How long is too long before a change is breaking? ;)